src/main/java/com/amazon/ion/impl/bin/FlexInt.kt

```diff
@@ -13,6 +13,11 @@ import java.math.BigInteger
  */
 object FlexInt {
 
+    /**
+     * A byte representing zero, encoded as a FlexInt (or FlexUInt).
+     */
+    const val ZERO: Byte = 0x01
```

Comment:
I introduced this because when we know that we're writing a FlexInt 0 (e.g. for a FlexSym), we may as well skip the conversion logic and write this byte directly.